cmd/root.go

```diff
@@ -0,0 +1,53 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	homedir "github.com/mitchellh/go-homedir"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var (
+	cfgFile string
+)
+
+var rootCmd = &cobra.Command{
+	Use:   "ultralist",
+	Short: "Ultralist, simple task management for tech folks.",
+}
+
+// Execute adds all child commands to the root command and sets flags appropriately.
+func Execute() {
+	if err := rootCmd.Execute(); err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+}
+
+func init() {
+	// cobra.OnInitialize(initConfig)
```

Comment:
Not required right now. Kept it for the case if we'd like to make ultralist configurable (e.g. colors).

Reply:
that will be the next step.  the [color](https://github.com/fatih/color) package has a `NoColor` flag that we can hook into, and will solve a lot of problems for windows users.